autoload/denops/plugin.vim

```diff
@@ -1,3 +1,34 @@
+let s:loaded_plugins = {}
+let s:load_callbacks = {}
+
+function! denops#plugin#is_loaded(name) abort
+  return has_key(s:loaded_plugins, a:name)
+endfunction
+
+function! denops#plugin#wait(name, ...) abort
+  if has_key(s:loaded_plugins, a:name)
+    return
+  endif
+  let options = extend({
+        \ 'interval': g:denops#plugin#wait_interval,
+        \}, a:0 ? a:1 : {},
+        \)
+  let expr = printf('sleep %dm', options.interval)
+  while !has_key(s:loaded_plugins, a:name)
+    execute expr
+  endwhile
+endfunction
+
+function! denops#plugin#wait_async(name, callback) abort
+  if has_key(s:loaded_plugins, a:name)
+    call a:callback()
+    return
+  endif
+  let callbacks = get(s:loaded_callbacks, a:name, [])
```

Comment:
Cause `E121: Undefied variable: s:loaded_callbacks` in this line.
Maybe it `s:load_callbacks` ?

Reply:
Sorry, missing information.
This problem caused when first execution of `denops#plugin#wait_async`.

Reply:
Fixed :bow: